README.md

```diff
@@ -1,13 +1,25 @@
 # Project Happy Thoughts API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This is the backend of an app called Happy thoughts. The app allows you to post a thought and like the existing thoughts. I have built the frontend in an earlier project. The purpose of this project was to practise setting up an API that allows users to POST data to a database in MongoDB. It is built using Node.js with Express and MongoDB with Mongoose.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I used Mongoose methods from the lectures. The endpoints worked when I tried them locally. Then I had trouble deploying. I got an error trying to look at the Heroku endpoint and I didn't see any collection in my MongoDB Atlas cluster. After much trouble shooting together with a team mate, it turned out I shouldn't have quotes around my connection string in the config vars in Heroku settings. Removing them, it worked! I Also had a little trouble getting the like feature to work when I connected this backend to my frontend. But it turned out there was a missing slash in the endpoint URL.
```

Comment:
Good that you solved these issues, but it's not necessary to go into detail here, especially as it was just some typos that caused the problem.